internal/uniqueness/leaderelection.go

```diff
@@ -29,24 +29,25 @@ import (
 	"time"
 
 	"github.com/elastic/elastic-agent-autodiscover/kubernetes"
-	"github.com/elastic/elastic-agent-libs/logp"
 	"github.com/gofrs/uuid"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1" // revive:disable-line
```

Comment:
`v1` is added by `goimports` but is considered redundant by revive. didn't find how to get around this other then disable review for the line